config/jetpack-cloud-stage.json

```diff
@@ -78,6 +78,7 @@
 		"jetpack-cloud-plugin-management": true,
 		"jetpack-cloud-pricing": true,
 		"jetpack-cloud-settings": true,
+		"jetpack-cloud-golden-token": false,
```

Comment:
I'd actually make this true so that it can be shown off in staging immediately.

Reply:
Will do!